CRM/Contribute/Form/AdditionalPayment.php

```diff
@@ -502,20 +461,24 @@ public function processCreditCard($submittedValues) {
     $paymentParams['contactID'] = $this->_contactId;
     CRM_Core_Payment_Form::mapParams($this->_bltID, $this->_params, $paymentParams, TRUE);
 
+    // add some financial type details (as contributionType_name) to the params list
```

Comment:
hmm - in a deprecated kind of a way - if it wasn't working from this form before & no-one has reported it as a problem I don't think we should add it